common/src/main/java/tds/common/cache/CacheConfigurationProperties.java

```diff
@@ -0,0 +1,55 @@
+package tds.common.cache;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ *
+ */
+@ConfigurationProperties(prefix = "tds.cache")
+public class CacheConfigurationProperties {
+    private int expireTimeShort = 20;
+    private int expireTimeMedium = 600;
+    private int expireTimeLong = 7200;
```

Comment:
Move configuration values into a ConfigurationProperties class and renamed these configuration properties slightly from:
```
tds.cache.expire.time.short => tds.cache.expireTimeShort
```